index.bs

```diff
@@ -335,9 +342,23 @@ Note: [Feature Policy](https://docs.google.com/document/d/1k0Ua-ZWlM_PsFCFdLMa8k
 should allow securely relaxing those restrictions once it matures.
 
 
+<h4 id="loosing-focus">Loosing Focus to a Different Origin</h4>
+
+When a different [=top-level browsing context=] of a different [=origin=] [=gains focus=],
```

Comment:
Is there an operation for that in HTML spec? Looks strange, implies that one top context knows origin of another context.